Doc/library/functools.rst

```diff
@@ -208,6 +208,55 @@ The :mod:`functools` module defines the following functions:
       Returning NotImplemented from the underlying comparison function for
       unrecognised types is now supported.
 
+.. function:: topsort(pairs)
+   Topologically sort a list of (parent, child) pairs. Return a sequence of
+   the elements in dependency order (parent to child order).
```

Comment:
This interface can't be used if your dependency graph has an isolated vertex (that is, a vertex with no edges), because there is no way to pass the isolated vertex to the function.

Reply:
Let's keep the discussion about the API in the issue. Although I agree with that, the limitation is on purpose. Different API's with different properties can be implemented and I am happy to discuss and implement the one that everyone agrees on.

Be aware that a lot of people can be confused if you process into a linear order isolated nodes.